cube/swiss/source/images/buttons/btns.h

```diff
@@ -17,8 +17,8 @@
 #define BTNEXIT_WIDTH   (100)
 #define BTNEXIT_HEIGHT  (38)
 
-#define BTNHILIGHT_WIDTH   (36)
-#define BTNHILIGHT_HEIGHT  (38)
+#define BTNHILIGHT_WIDTH   (67)
+#define BTNHILIGHT_HEIGHT  (49)

```

Comment:
Textures must be a multiple of 4x4.

Reply:
I've adjusted the highlight button texture to 4x4 multiple